### ISSUE TITLE
Copy world URL button in world menu copies URL which returns 403

### ISSUE THREAD

**Issue by eai04191** [Yellow-Dog-Man/Resonite-Issues#160]
### Describe the bug?

For example, copying the **Resonite Cloud Home** world URL will copy the following URL:
```
https://go.resonite.com/record/G-Resonite-Team/R-c3f6fd32-d82f-4f48-80f1-87a05b184650
```

This page has two problems
- HTTPS certificate does not match domain
- It's returning status code 403 and doesn't work

### To Reproduce

1. Open the world search screen
2. Search for "Resonite Cloud Home"
3. Find “Resonite Cloud Home” by “Resonite Team”
    (To make it easier to find, open **Custom** and check **Only Featured**)
4. Click to open world information
5. Select "Copy World URL"
6. This URL has the above problem

### Expected behavior

It should return a world URL that you can use.

### Resonite Version Number

2023.10.13.743

### What Platforms does this occur on?

Windows

### What headset if any do you use?

Desktop

### Reporters

- IGN: Eai
- Discord: `<@132779412771962880>` (`@eai`)

**Comment by Frooxius** [comment]
Is the world you're copying the URL for set to Public? Not the session itself, but the world in the metadata.

403 means that you're not allowed to access it, which means you're making un-authenticated request.

**Comment by eai04191** [comment]
Maybe? I understand that “Resonite Cloud Home” by “Resonite Team” is a public world.
The steps to reproduce it were unclear, so I added it.

**Comment by eai04191** [comment]
Something I noticed today: The Copy World URL button in the session menu returns a different URL.
It's a URL like this:
```
https://api.resonite.com/open/world/G-Resonite-Team/R-c3f6fd32-d82f-4f48-80f1-87a05b184650
```
Unlike go.resonite.com, this URL appears to 302 redirect to the following `resonite:` protocol URL without any certificate issues:
```
resonite:?world=resrec:///G-Resonite-Team/R-c3f6fd32-d82f-4f48-80f1-87a05b184650
```

However, nothing happened on the game side when I opened this. Either the game is not recognized by the OS as a handler for the `resonite:` protocol, or something else, but it seems to be incomplete.
